docs/dev-machine-setup.md

```diff
@@ -175,18 +160,17 @@ set up a Ubuntu-based VM on GCE with nested KVM enablement can be found in GCE
     uploaded to GCE. Done! You should see the prompt of the new VM:
 
     ```console
-    ubuntu@firecracker-vm:~$
+    [YOUR_USER_NAME]@firecracker-vm:~$
     ```
 
 1. Verify that VMX is enabled, enable KVM
 
     ```console
     $ grep -cw vmx /proc/cpuinfo
     1
-    $ sudo setfacl -m u:${USER}:rw /dev/kvm
-    $ [ -r /dev/kvm ] && [ -w /dev/kvm ] && echo "OK" || echo "FAIL"
-    OK
```

Comment:
May I ask why you removed these lines?

The grep command does a hardware-level check and ensures that the processor has the hardware-assisted virtualization feature, while the rest commands that you deleted check software-level capability of KVM.

Reply:
I removed it since i didn't get it to work and though the check was not necessary. But understand it really make sens to keep!